tower-http/src/services/fs/serve_dir.rs

```diff
@@ -120,6 +159,16 @@ impl ServeDir {
         self.buf_chunk_size = chunk_size;
         self
     }
+
+    /// Cache-Control:max-age=300.
+    ///
+    /// This speeds up user's feeling for static sites, especially large files.
+    ///
+    /// Defaults to 300 seconds.
+    pub fn max_age(mut self, dur: Duration) -> Self {
+        self.max_age = dur;
+        self
+    }
```

Comment:
I think it makes sense to allow users to set this to `None` if for some reason they don't wanna specify a `max-age`. Could also set it to `Duration::ZERO` but I think an explicit `Option<Duration>` is more clear.

```suggestion
    pub fn max_age(mut self, max_age: Option<Duration>) -> Self {
        self.max_age = max_age;
        self
    }
```

This'll require some more changes elsewhere.